plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -127,7 +127,7 @@ public function enqueue_scripts() {
 		$editor_settings = $this->get_product_editor_settings();
 
 		$script_handle = 'wc-admin-edit-product';
-		wp_register_script( $script_handle, '', array(), '0.1.0', true );
+		wp_register_script( $script_handle, '', array( 'wp-blocks' ), '0.1.0', true );
```

Comment:
I noticed an error that `setCategories` was not defined for the inline script, this addresses that.